hikari/internal/ux.py

```diff
@@ -370,22 +373,21 @@ async def check_for_updates(http_settings: config.HTTPSettings, proxy_settings:
 
         this_version = HikariVersion(about.__version__)
         is_dev = this_version.prerelease is not None
-        newer_releases: typing.List[HikariVersion] = []
+        newest_version: typing.Optional[HikariVersion] = None
```

Comment:
> Does this actually change the behaviour at all or is this just a surplus style change?

It uses less memory and is a bit more efficient, since we only ever store what the version we are going to print is instead of storing all possible versions in a list and then sending the newest one

> Also you need to check for yanked releases

Yanked releases are already checked bellow.